darwin/DarwinProcessList.c

```diff
@@ -22,51 +22,10 @@ in the source distribution for its full text.
 #include "ProcessList.h"
 #include "darwin/DarwinProcess.h"
 #include "darwin/Platform.h"
+#include "darwin/PlatformHelpers.h"
 #include "generic/openzfs_sysctl.h"
 #include "zfs/ZfsArcStats.h"
 
```

Comment:
Should be two blank lines here too …

Reply:
It was already like that, but yes, I'll fix it :)